### ISSUE TITLE
TS types are wrong for <FlatList _contentContainerStyle>

### ISSUE THREAD

**Issue by garrett-gottlieb** [GeekyAnts/NativeBase#4797]
### Description

_contentContainerStyle requires data and renderItem properties, when it should only require style props

### CodeSandbox/Snack link

none

### Steps to reproduce

```ts
<FlatList
  data={data}
  renderItem={renderItem}
  _contentContainerStyle={{ pb: '8' }} // TS ERROR
  // ...

### NativeBase Version

^3.3.7

### Platform

- [ ] Android
- [ ] CRA
- [X] Expo
- [X] iOS
- [ ] Next

### Other Platform

TypeScript

### Additional Information

_No response_

**Comment by garrett-gottlieb** [comment]
Looks like there's a PR to fix it that hasn't been merged: https://github.com/GeekyAnts/NativeBase/pull/4642

**Comment by surajahmed** [comment]
@garrett-gottlieb Thanks for working on this. We've fixed this issue in version `3.3.9`.
